src/contracts/pods/EigenPod.sol

```diff
@@ -792,5 +928,5 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[44] private __gap;
+    uint256[41] private __gap;
```

Comment:
rn I think this size is off by 1
I count 4 new slots used with the gap decreasing in size by 3

Reply:
ah fixed, didn't update it after I added  lastRequestNonceProven;